lib/normalize-query-options/offset.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Normalize the offset option. If no limit is defined, then return offset as undefined. ala-sql
+ * requires OFFSET to be paired with a LIMIT
+ * @param {object} options
+ * @returns {integer} or undefined
+ */
+function normalizeOffset (options) {
+  return (options.limit) ? (options.offset || options.resultOffset) : undefined
```

Comment:
same for limit=0?

Reply:
Good catch.  This is the old code and just putting under test, but I'll make an adjustment to allow for 0 limit.